src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -9,8 +9,6 @@
     <DefineConstants>$(DefineConstants);FEATURE_CORECLR</DefineConstants>
   </PropertyGroup>
   <!-- Help VS understand available configurations -->
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FreeBSD_Debug|AnyCPU' " />
```

Comment:
I'm pretty sure that we shouldn't have configurations that don't build in our project files. This configuration can be re-added when the community adds support for FreeBSD to S.N.Sockets.